pkgs/development/python-modules/zarr/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools_scm
+, asciitree
+, numpy
+, fasteners
+, numcodecs
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "zarr";
+  version = "2.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "c62d0158fb287151c978904935a177b3d2d318dea3057cfbeac8541915dfa105";
+  };
+
+  nativeBuildInputs = [
+    setuptools_scm
+  ];
+
+  propagatedBuildInputs = [
+    asciitree
+    numpy
+    fasteners
+    numcodecs
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
```

i couldn't get this to compile correctly without:
```suggestion
  checkPhase = ''
    pytest . -k 'not storage'
  '';
  
```

---

Yeah I'm close to getting this PR to fully work. The issue is that zarr didn't properly remove the azure tests and its thowing errors there. Trying to find a nice workaround

---

This disables those, just try it :)

---

Got it working thanks. I found that I didn't have to disable any tests. But that I needed to explicetly call `pytest`

---

really? mine didn't... weird

---

Are you running on darwin?

---

NixOS

---

Hmm same here

---

it builds fine now. Weird